azext_iot/common/shared.py

```diff
@@ -11,6 +11,11 @@
 
 from enum import Enum
 
+# Open certificate validation strings
+INVALID_BASE64 = "The certificate content is not a valid base64 string value"
```

Comment:
We (as a team) should chat later about organizing strings like these, but I'm fine for a temp placement.

Reply:
my only nit here is this needs a .

Since this is only used in the open certificate tests, you can word it as:
"The certificate file does not have a valid base64 string value."